app/views/user/contribute/_completed_tracks.erb

```diff
@@ -0,0 +1,19 @@
+<h4>You have completed the current tracks, you can contribute by adding a new exercise from another language track, below you will find the link for each finished track</h4>
+<table class="table table-bordered table-striped" data-sortable>
+  <col width="25%" />
+  <thead>
+    <tr>
+      <th data-sorted="true"  data-sorted-direction="ascending">Completed Tracks</th>
+      <th>Link</th>
+    </tr>
+  </thead>
+  <tbody>
+  <% contribute_info.each do |hash| %>
+    <tr>
+      <td><%= hash['language_name'] %></td>
```

Comment:
There's a lot of confusion throughout the codebase right now, which I intend to clean up.

"Language" should refer to the actual language name C#, F#, JavaScript, ECMAScript.
"Track ID" should refer to our url-friendly version of the language name, used to identify the language track on Exercism.